shared/src/main/scala/coop/rchain/catscontrib/effect/implicits/package.scala

```diff
@@ -3,15 +3,34 @@ package coop.rchain.catscontrib.effect
 import cats._
 import cats.effect.ExitCase.{Completed, Error}
 import cats.effect._
-import scala.concurrent.ExecutionContext
 
-import scala.concurrent.duration.Duration
-import scala.concurrent.{Await, Promise}
 import scala.util.{Failure, Success, Try}
 import scala.util.control.NonFatal
 
 package object implicits {
 
+  // this is for testing purposes, do not use in production code!
```

Comment:
This has to be defined somewhere under `test`, otherwise it will be used (Murphy's law)

Reply:
it's required by Address Examples

Reply:
Then their fate is sealed. After merging this PR, we should draw lots who's gonna take the honors to remove them, along with that instance. Unless you'd like to have that privilege, @dzajkowski? (I think you deserve that glory :))

Reply:
go for it.